cetmix_tower_server/views/cx_tower_variable_option_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="cx_tower_variable_option_form" model="ir.ui.view">
+        <field name="name">cx.tower.variable.option.form</field>
+        <field name="model">cx.tower.variable.option</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <group>
+                        <field name="name" string="Option Value" />
```

Comment:
No need to add 'string' here as long as it's the same as the field string defined in the model.

Reply:
I will fix it